Makefile

```diff
@@ -5,6 +5,11 @@ BUILD=build
 BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
 BIN_DIR?=.
 
+BUILD_TIME=$(shell date +%s)
```

Comment:
Missing these variables in `cmd/dp-search-api/main.go`

If you add the following to `main.go`:
```
var (
        // BuildTime represents the time in which the service was built
        BuildTime string
        // GitCommit represents the commit (SHA-1) hash of the service that is running
        GitCommit string
        // Version represents the version of the service that is running
        Version string
)
```